internal/index/indexscanner/scanner_test.go

```diff
@@ -90,6 +94,58 @@ func TestReadPluginFile(t *testing.T) {
 	}
 }
 
+func TestReadReceiptFromFile(t *testing.T) {
+	type args struct {
+		receiptFilePath string
+	}
+	tests := []struct {
+		name       string
+		args       args
```

Comment:
I haven't read the previous review comments, but is there a reason why `receiptFilePath` is wrapped in this args struct? There could as well just be a single `receiptFilePath` field.

Reply:
Its left over from a previous version. I had some other things there before that I moved out, I'll make the change.